en/config_vtol/vtol-back-transition-tuning.md

```diff
@@ -0,0 +1,36 @@
+# VTOL back transition tuning
+
+> **Tip** Some of the following features will be available in PX4 version 1.7 and are currently only available on the experimental development branch.
```

Comment:
Perhaps change to a "Note". This is more like a warning, and the Tip is a bit too friendly for something that might confuse people.